### ISSUE TITLE
Adding support for Bhojpuri, Magahi and Maithili

### ISSUE THREAD

**Issue by singhakr** [goru001/inltk#89]
We would like to release some data and possible some tools for these three languages. How can we do it?

- Anil Kumar Singh

**Comment by goru001** [comment]
Thanks for reaching out @singhakr - Do you already have data corpus for these languages?

**Comment by singhakr** [comment]
Yes, we have plain text as well as POS tagged and chunked corpus and some
other resources.

**Comment by goru001** [comment]
Thats great @singhakr . Do you want to start by following pointers in this [thread](https://github.com/goru001/inltk/issues/1#issuecomment-477857250)?

**Comment by singhakr** [comment]
Yes, I would.
